lib/scout_apm/utils/sql_sanitizer_regex.rb

```diff
@@ -10,6 +10,7 @@ module SqlRegex
       PSQL_REMOVE_INTEGERS = /(?<!LIMIT )\b\d+\b/.freeze
       PSQL_PLACEHOLDER = /\$\d+/.freeze
       PSQL_IN_CLAUSE = /IN\s+\(\?[^\)]*\)/.freeze
+      PSQL_AFTER_WHERE = /(?<=where ).*$/i.freeze
```

Comment:
I don't think the lookbehind is needed in the regex, just adds complexity. Since it just gets sent through the string sanitizer anyway, which will never match the 'WHERE' part.